scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -26,6 +26,12 @@ namespace <%= namespace %> {
             }
         }
 
+        public List<UserError> UserErrors {
```

Comment:
I've been thinking about this, and I'm wondering have you considered subclassing `ShopifyError` -> `ShopifyUserError`, that includes this list of UserErrors?

`UserErrors` can become out of sync on the fly, if implementor is always accessing it like so `Cart.UserErrors` they might catch `UserErrors` from another Query.